### PR TITLE
Added jqwidgets TypeScript definitions

### PR DESCRIPTION
**Pull request by DimitarD**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by orta** [comment]
Hi, given that this comes form the jqwidgets team is there a reason why the d.ts files are not shipped with the library and then will be accurate to the version on npm?

**Comment by DimitarD** [comment]
The d.ts file is actually included in our npm package (jqwidgets-scripts). However, we would like for it to be featured here too, this repository being an exhaustive list of TypeScript type definitions.

File: types/jqwidgets/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,4896 @@
+// Type definitions for jqwidgets-scripts
```
Comment (DanielRosenwasser) [inline types/jqwidgets/index.d.ts]: Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to be a `MAJOR.MINOR` version (e.g. `8.3`)?

Also, it looks like this containing folder was incorrectly named. jqwidgets doesn't exist on npm, but jqwidgets-scripts does.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
> The d.ts file is actually included in our npm package (jqwidgets-scripts). However, we would like for it to be featured here too, this repository being an exhaustive list of TypeScript type definition

I see; however, that's not the purpose of DefinitelyTyped. DefinitelyTyped exists to fill the gap for packages that don't contain `.d.ts` files. Thanks for taking the time though.

**Pull request closed without merging**